src/components/FieldNumber/FieldNumber.js

```diff
@@ -30,6 +30,7 @@ const IconMinus = props => {
       viewBox="0 0 40 40"
       fill="none"
       xmlns="http://www.w3.org/2000/svg"
+      role="none"
```

Comment:
These are buttons with only icon inside. I think you should add aria-label for these buttons (there seems to be a title attribute already). That way the screenreaders have a standard way to read something relevant for the button.
(Note: title attribute might be a fallback for some readers, but since we are now in the business of adding aria labels, that's a better solution I think)